### ISSUE TITLE
Some questions about data processing

### ISSUE THREAD

**Issue by ardiantovn** [doggydoggy0101/recommendation_network#1]
1. In `ml_latest.ipynb` file, why is `input_watch_history` normalized? isn't `input_watch_hist `an ordinal data that doesn't need to be normalized?
```
# Embedding layer 
features_embedding_layer=Embedding(input_dim=NUM_CLASSES,output_dim=EMBEDDING_DIMS,mask_zero=True,trainable=True,name='features_embeddings')
features_embeddings = features_embedding_layer(input_watch_hist)
l2_norm_features = l2_norm_1(features_embeddings)
avg_features = avg_embeddings(l2_norm_features)
```

2. Why is the input `train_data` added with `+ 1e-10` ?
```
history=model.fit([pad_sequences(train_data['watch_hist'])+ 1e-10,
                  pad_sequences(train_data['example_age'],dtype=float)+ 1e-10,
                  ],
                  pad_sequences(train_data['labels']),
                  batch_size=128, 
                  validation_split=0.1, 
                  epochs=EPOCH,
                  verbose=1)
```

3. Why is the `input_example_age` embedded on `labels_embedding_layer` and being averaged? 
```
labels_embedding_layer=Embedding(input_dim=NUM_CLASSES,output_dim=EMBEDDING_DIMS,mask_zero=True,trainable=True,name='labels_embeddings')
labels_example_age_embeddings = labels_embedding_layer(input_example_age)
l2_norm_example_age = l2_norm_1(labels_example_age_embeddings)
avg_example_age = avg_embeddings(l2_norm_example_age)
```

Based on this picture, I thought `input_example_age` (a continuous data) is not embedded and averaged. CMIIW

<img width="517" alt="Screenshot 2022-11-03 at 10 28 19" src="https://user-images.githubusercontent.com/16162415/199642340-fdc067bf-b182-40ea-b0e3-887ef8e78e28.png">

Thank you
